### PR TITLE
[FIX] point_of_sale: ticket cashier does not change after order finished

### PR DESCRIPTION
**Pull request by Ephoniz**
Steps to reproduce:

- Install PoS and activate Multi Employees per Session (Add atleast 2 employees)
- Open a session in the PoS shop and create a new order.
- Follow the steps until the ticket screen of the order.
- Here change the cashier of the PoS, and see the ticket cashier.

We don't want to change the ticket cashier of an order that has already been done, since then the cashier of the order is not going to match the cashier of the ticket.

opw-3483405

### CODE REVIEW

**Review by robinengels — COMMENTED** [review]
LGTM.
@caburj can you r+ ?

**Comment by Ephoniz** [comment]
@caburj up 😄

File: addons/point_of_sale/static/src/js/models.js [hunk 1]
```diff
@@ -2524,7 +2524,7 @@ class Order extends PosModel {
                 return paymentline.export_for_printing();
             });
         let partner = this.partner;
-        let cashier = this.pos.get_cashier();
+        let cashier = this.pos.get_order().cashier;
```
Comment (caburj) [inline addons/point_of_sale/static/src/js/models.js:2527]:
```suggestion
        let cashier = this.cashier;
```

The method is already in the Order class, so take the cashier in the instance, not from the current order.

Comment (Ephoniz) [reply addons/point_of_sale/static/src/js/models.js]: Oops, Thanks for the review! 😄

**Comment by caburj** [comment]
I'd like to merge but it's red. Maybe rebase?

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**